hamilton/function_modifiers/metadata.py

```diff
@@ -57,6 +57,7 @@ def my_function(...) -> ...:
         "ccpa",
         "dag",
         "module",
+        "ERROR_SENTINEL",
```

Comment:
I don't think this is a namespace? I'd set it as `hamilton.error_sentinel=True)` then keep the ypassing of reserved namespaces. That said, we can always change it later as its an internal contract...

Reply:
[96d9c47](https://github.com/DAGWorks-Inc/hamilton/pull/1017/commits/96d9c47d9558529668ec30a78721baa9dd230794)